### PR TITLE
fix enter press bug in button component

### PR DESCRIPTION
**Pull request by slee1130**
### Purpose 🚀
_clear, one sentence description of primary purpose of this PR_

### Notes ✏️
_details of code change / secondary purposes of this PR_
- Press space or use mouse to click the button, it shows a subtle indent in the button to show user feedback.
-Resolves https://github.com/acl-services/paprika/issues/299

@oscarkwan 

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [ ] MINOR (backward compatible) change to _these packages_
- [x] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/Button/src/Button.js [hunk 1]
```diff
@@ -136,10 +137,25 @@ const Button = React.forwardRef((props, ref) => {
     if (!isButtonDisabled) onClick(event);
   };
 
+  const handleKeyDown = event => {
+    if (event.key === "Enter") {
+      console.log(event.currentTarget);
```
Comment (oscarkwan) [inline packages/Button/src/Button.js]: remove :)

Comment (slee1130) [reply packages/Button/src/Button.js]: oops sorry..! done now :)

**Comment by alexzherdev** [comment]
@oscarkwan https://github.com/acl-services/paprika/issues/299#issuecomment-560948854

File: packages/Button/src/Button.js [hunk 2]
```diff
@@ -136,10 +137,24 @@ const Button = React.forwardRef((props, ref) => {
     if (!isButtonDisabled) onClick(event);
   };
 
+  const handleKeyDown = event => {
```
Comment (nahumzs) [inline packages/Button/src/Button.js]: any reason why this is not done via style components? cc @oscarkwan

**Comment by oscarkwan** [comment]
Hey all, closing this PR! Confirmed with mike and sarah. We should just follow the native way.

**Pull request closed without merging**